tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/home.controller.js

```diff
@@ -190,9 +190,21 @@
                 }
 
                 if (data != "Settings") {
-                    $window.top.parent.location.href = configs.uri.SPOsiteURL + "/SitePages/MatterCenterHome.aspx?" + data;
+                    if (configs.global.isBackwardCompatible == false) {

```

Comment:
If the settings of isBackwardCompatible  is true, the sharepoint url will be different